error_test.go

```diff
@@ -56,6 +58,9 @@ func TestErrorMessages(t *testing.T) {
 		"arr_oob_nested_wo_meta": raiseIndexOutOfBounds(arrNested, 5),
 		"arr_oob_nested_w_meta":  raiseIndexOutOfBounds(arrNestedMeta, 5),
 
+		"invalid_duration": raiseInvalidDuration(newString(
```

Comment:
testMeta is adding the `(source: ...)` to the message. Maybe introduce 2 tests (`invalid_duration_wo_meta` and `invalid_duration_w_meta`)